utils/oscap-remediate-offline-service

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2021 Red Hat Inc., Durham, North Carolina.
+#
+# Evgenii Kolesnikov <ekolesni@redhat.com>
+#
+# The offline system remediation (systemd's /system-update) service wrapper
+
+log() {
+    echo "${1}" | logger -t oscap-remediate-offline -p ${2:-info}
+}
+
+config=$(readlink -f /system-update)
+
+if [[ "$(basename ${config})" != "oscap-remediate-offline.conf.sh" ]]; then
+    log "The /system-update symlink was not created by oscap-remediate-offline, ignoring"
+    systemd-notify --ready --status="Skipped"
+    exit 0
+fi
+
+rm -f /system-update
+
+log "Found config: ${config}"
+
+source "$config"
+
+log "Data stream: ${OSCAP_REMEDIATE_DS}"
+log "Profile ID: ${OSCAP_REMEDIATE_PROFILE_ID}"
+
+#TODO: Check
+
+profile_title_line=$(oscap info --profile "${OSCAP_REMEDIATE_PROFILE_ID}" "${OSCAP_REMEDIATE_DS}" | grep Title)
+
+#TODO: Check
+
+profile_title=${profile_title_line#*Title: }
+
+log "Profile name: ${profile_title}"
+
+oscap xccdf validate --skip-schematron "${OSCAP_REMEDIATE_DS}"
+
+#TODO: Check
+
+log_dir="/var/lib/ssg/$(date +%Y.%m.%d-%H:%M:%S)"
+
+mkdir -p "${log_dir}" && cd "${log_dir}"
+
+#TODO: Check?
+
+command="oscap xccdf eval --profile ${OSCAP_REMEDIATE_PROFILE_ID}\
+    --results xccdf-results.xml\
+    --results-arf arf-results.xml\
```

Comment:
Hmm. Okay, makes sense.

Reply:
I've removed the `--results` argument. But now I see that OAA uses it instead of `--results-arf` and Workbench has them both for some reason. @matejak @jan-cerny Do we need it or not? Should it be optional?

Reply:
I would say that we don't need it because it contains the same data that are present in the ARF file.

Regarding the OAA and Workbench I think they use --results just for a historic reason because in very old versions oscap didn't support ARF.